### ISSUE TITLE
Migrate to PlatformWidgets

### ISSUE THREAD

**Issue by giannissc** [matthinc/flutter_cupertino_settings#8]
First I would like to thank for putting the time to create this library. It really has helped me speed up the completion of one of my assignments

After playing around with the example and going through your code I saw that it would be relatively easy to switch form using native Cupertino widget to using Platform Widgets and make this a generic settings library for both Android and iOS

https://github.com/aqwert/flutter_platform_widgets

I am be willing to contribute and pull the changes myself but that will happen either mid april or after the middle of may.

**Comment by matthinc** [comment]
Yes, that sounds like a good idea. I'll keep this in mind, when I have time again to work on this project.
